src/data/faq.json

```diff
@@ -101,6 +101,166 @@
+                    {
+                        "title": "Can I show the QR code without an internet connection?",
+                        "anchor": "vaccination_certificate_internet",
+                        "active": true,
+                        "textblock": [
+                            "Yes, indeed. As soon as the QR code is displayed in the Corona-Warn-App, you can show it at any time without an existing internet connection."
+                        ]
+                    },
+                    {
+                        "title": "Am I obliged to use the digital certificate of vaccination in the Corona-Warn-App? Will there still be the yellow analog vaccination booklet? Can I continue to use my yellow vaccination booklet? What about people who don't own a smartphone?",
+                        "anchor": "vaccination_certificate_voluntary",
+                        "active": true,
+                        "textblock": [
+                            "The digital proof of vaccination via the Corona-Warn-App is a completely voluntary and complementary offer. If you do not have a digital certificate of vaccination or have lost it, the proof of vaccination via the well-known 'yellow booklet' (yellow vaccination passport) is still possible and valid. You can also prove Corona vaccinations with the printed Corona vaccination certificate for full vaccination."                        ]
+                    },
+                    {
+                        "title": "How is the proof of vaccination verified?",
+                        "anchor": "vaccination_certificate_verification",
+                        "active": true,
+                        "textblock": [
+                            "The vaccination certificate (QR code) is used digitally via the CovPass-App or the Corona-Warn-App (CWA) or alternatively as a machine-readable print. The vaccination certificate contains only information about the vaccination status, the name of the vaccinated person, and the date of birth. For service providers who want to check the vaccination status, there will be a verification app to check the vaccination certificate. This allows the vaccination status to be scanned in a similar way to a barcode of a flight or train ticket. Alternatively, proof with the analogue vaccination document is possible."
+                        ]
+                    },
+                    {
+                        "title": "Can you also store in the digital vaccination certificate that you have already been infected or tested negative?",
+                        "anchor": "vaccination_certificate_recovered_negative_test",
+                        "active": true,
+                        "textblock": [
+                            "Yes, negative tests or a recovery from the Corona virus can also be stored in the CovPass-App and Corona-Warn-App as a test certificate or recovery certificate. A recovery from a Corona infection is documented by means of a PCR test result."

```

Comment:
Is this true for the CWA 2.3 release already?

Reply:
This has to be cross-checked. I think it will be not there.

Reply:
remove vac_cert_recovered_negative_test